pages/en/lb3/Attaching-models-to-data-sources.md

```diff
@@ -132,6 +149,14 @@ you can specify the data source you want it to use from among those you've added
 [Data source generator](Data-source-generator.html) and the default `db`
 data source (that uses the [memory connector](Memory-connector.html)).
 
+```shell
+? Enter the model name: myModel
```

Comment:
There is actually a link for the model generator 2 lines above. I am just trying to explain to the user that you don't need to do it manually but you can attach the model using the command prompt. I am more concerned here about the model attachment.